### PR TITLE
added cuttlefish_util:conf_get_value/3

### PR DESCRIPTION
**Pull request by joedevivo**
This function was supposed to be a convenience function for schema writers that wrapped proplists:get_value

It only makes sense to have the arty 3 version

### CODE REVIEW

File: src/cuttlefish_util.erl [hunk 1]
```diff
@@ -47,6 +48,13 @@ conf_get_value([H|_T]=Variable, ConfigProplist) when is_list(H) ->
 conf_get_value(Variable, ConfigProplist) ->
```
Comment (seancribbs) [inline src/cuttlefish_util.erl:46]: Why not simplify this by having the 2-arity version call the 3-arity version with the third argument of `undefined`.

Comment (joedevivo) [inline src/cuttlefish_util.erl:46]: ha, that almost seems "too" correct. will do.

**Comment by seancribbs** [comment]
Dialyzed this, found some bugs. These are not part of the PR, but should be addressed soon. Ideally rebar would be added to the PLT to resolve the mustache/rebar problems, but I'm not quite sure how to do that without making rebar a dependency.

```
Unknown functions:
  eunit:test/1
  inti:stop/1
  mustache:render/2
  rebar_rel_utils:get_target_dir/2
  rebar_rel_utils:is_rel_dir/0
  rebar_rel_utils:load_config/2
```

```
$ ack "inti:stop" src
src/cuttlefish_escript.erl
105:            inti:stop(1);
```

**Comment by seancribbs** [comment]
Disregard the `inti:stop` error, it is not present on `develop`.

Rest of this is :+1: to merge.

**Pull request merged**